packages/itwin/grouping-mapping-widget/src/widget/components/CalculatedPropertyActionWithVisuals.tsx

```diff
@@ -143,6 +187,16 @@ export const CalculatedPropertyActionWithVisuals = ({
             type,
           },
         );
+      const promises: any[] = [];
```

Comment:
See if you can avoid an `any` here.

Reply:
Tried to avoid using the "any" here. But I am unable to find a work around for this at the moment.

Reply:
`Promise<void>[]`

Reply:
Made changes to address this.